.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: "\U0001F41B Bug report"
+about: Create a report to help us improve
```

Comment:
nit: `Submit a bug report to help us improve LLM Engine. If this doesn't look right, [choose a different type.](https://github.com/scaleapi/llm-engine/issues/new/choose)`